frontend/src/app/main/ui/dashboard/sidebar.scss

```diff
@@ -392,17 +392,85 @@
 
 .profile-dropdown {
   @extend .menu-dropdown;
-  left: var(--sp-l);
-  bottom: deprecated.$s-72;
-  min-width: $sz-252;
+  inset-inline-start: var(--sp-s);
+  inset-block-end: deprecated.$s-72;
```

Comment:
Q: where does this 72px come from?

If it's from the design itself, we should create a new token in our `_sizes.scss` (and inform Design so they can update the design system)

If it's from some calculation, maybe we could add a comment, or use `calc` to be more explicit?